opensearch/src/main/java/org/opensearch/sql/opensearch/request/OpenSearchScrollRequest.java

```diff
@@ -34,7 +33,7 @@
 public class OpenSearchScrollRequest implements OpenSearchRequest {
 
   /** Default scroll context timeout in minutes. */
-  public static final TimeValue DEFAULT_SCROLL_TIMEOUT = TimeValue.timeValueMinutes(1L);
+  public static final TimeValue DEFAULT_SCROLL_TIMEOUT = TimeValue.timeValueMinutes(100L);
```

Comment:
I set it for debugging purposes, but didn't revert, because 1 min is too small.
TBD.
@penghuo, @dai-chen,
What should be the default scroll lifetime?
We're looking forward to make it configurable.

Reply:
@Yury-Fridlyand you mean this configuration? https://github.com/opensearch-project/sql/blob/main/docs/user/admin/settings.rst#plugins-sql-cursor-keep-alive